utils/shared/src/main/scala/fastparse/Utils.scala

```diff
@@ -1,12 +1,13 @@
 package fastparse
 
+import java.io.InputStream
+
 import scala.annotation.{switch, tailrec}
-import acyclic.file
+//import acyclic.file
```

Comment:
Looks like you had problems with the dependency-cycle-checker! Your problem is you are depending on `ParserInput` from `Utils.query`, but presumably also depending on `Utils` from `ParserInput`. The solution here is to make `ParserInput[T]` implement a smaller `IsReachable[T]` interface defined together with `Utils`, and make `query` only depend on the `IsReachable` interface